marshmallow/validate.py

```diff
@@ -465,3 +465,48 @@ def __call__(self, value):
                 del choices[index]
 
         return value
+
+
+class Or(Validator):
+    """Validator which succeeds if ``value`` is validated by at least one of
+    ``validator_list`` elements
+
+    :param validator_list:
+    :param str error: Error message to raise in case of a validation error. Can be
+        interpolated with `{input}`.
+    """
+
+    default_message = "{input!r} can't be validated by any of the validators"
+
+    def __init__(self, *validator_list, error=None, **kwargs):
+        if any(not isinstance(validator, Validator) for validator in
+                validator_list):
+            raise ValueError("This validator accepts only validators as "
+                             "parameters")
+        self.validator_list = validator_list
+        self.error = error or self.default_message
+        super(Or, self).__init__(**kwargs)
+
+    def __call__(self, value):
+        for validator in self.validator_list:
+            try:
+                validator(value)
+            except ValidationError:
+                pass
+            else:
+                return value
+
+        # All validators raised an error
+        raise ValidationError(self._format_error(value))
+
+    def _repr_args(self):
+        return "validator_list={!r}".format(self.validator_list)
+
+    def _format_error(self, value):
+        return self.error.format(input=value)
+
+    def add_validator(self, validator):
```

Comment:
I admit I don't *need* this method. I only thought that people *could* want to add validators, but if they do it by accessing the list attribute, the validator check wouldn't be done. But as you highlight, I don't think a lot of people use schema in this way. Shall I remove it ?